src/test/java/core/basesyntax/RegistrationServiceTest.java

```diff
@@ -0,0 +1,128 @@
+package core.basesyntax;
+
+import core.basesyntax.exception.RegistrationException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.Test;
+
+
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class RegistrationServiceTest {
+    private RegistrationService registrationService = new RegistrationServiceImpl();
```

Comment:
Consider using `@BeforeEach` or `@BeforeAll` to initialize your `RegistrationService` instance. `@BeforeEach` will create a new instance before each test, ensuring test isolation. `@BeforeAll` will create a single instance before all tests run, which is more efficient but can lead to shared state if not careful.